gdnative-core/src/core_types/mod.rs

```diff
@@ -43,7 +43,7 @@ pub use rid::*;
 pub use string::*;
 pub use string_array::*;
 pub use transform2d::*;
-pub use typed_array::TypedArray;
+pub use typed_array::{Element, TypedArray};
```

Comment:
Is this `pub use` necessary? The `typed_array` module is public so you should be able to import it if you want anyway.

Reply:
Oh, I kinda forgot about this. It was actually the first change I made because I was trying to implement this in my own crate and couldn't for `TypedArray`s. 

The `typed_array` module is actually *not* public. Should it be made so instead of re-exporting `Element`?

Reply:
Oh, I saw it show up on rustdoc so I assumed it was public. If it isn't, then by all means keep the re-export!